code/data.py

```diff
@@ -0,0 +1,15 @@
+import json
+import pandas as pd
+
+print("Converting data from json to csv...")
+
+""" JSON TO DATAFRAME """
+
+data = json.load(open('../data/device-covid19serology-0001-of-0001.json'))
+
+df = pd.DataFrame(data["results"])
+"""print(df)"""
+
+""" DATAFRAME TO CSV """
+
+df.to_csv('../data/device-covid19serology-0001-of-0001.csv', index = None)
```

Comment:
Do we need to convert the data to CSV? Probably the dataframes are better to query the data in order to get insights!

Reply:
Once we have the data in CSV format, it is very easy to import it into SQL and design the queries to obtain information. On the other hand, having the data in dataframe makes it easier to design visualisations (graphs) to show on the website. That's why I think it's quite appropriate to have the data accessible in these two ways.